packages/components/src/circular-option-picker/option-picker-option.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { useContext, useEffect } from '@wordpress/element';
+import { Icon, check } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { CircularOptionPickerContext } from './option-picker-context';
+import Button from '../button';
+import { CompositeItem } from '../composite';
+import Tooltip from '../tooltip';
+import type { OptionProps } from './types';
+
+const hasSelectedOption = new Map();
+
+function renderOptionAsButton( props: {
+	className?: string;
+	isPressed?: boolean;
+} ) {
+	return <Button { ...props }></Button>;
+}
+
+function renderOptionAsOption( props: {
```

Comment:
Any reason why these are render functions, instead of components?

Reply:
No reason in particular!